lib/sles4sap/azure_cli.pm

```diff
@@ -134,6 +134,11 @@ sub az_network_vnet_create {
         croak("Argument < $_ > missing") unless $args{$_}; }
     $args{address_prefixes} //= '192.168.0.0/16';
     $args{subnet_prefixes} //= '192.168.0.0/24';
+    foreach (qw(address_prefixes subnet_prefixes)) {
+        croak "Invalid IP range $args{$_} in $_"
+          unless ($args{$_} =~ /^[1-9]{1}[0-9]{0,2}\.(0|[1-9]{1,3})\.(0|[1-9]{1,3})\.(0|[1-9]{1,3})\/[0-9]+$/);
```

Comment:
There's is something that @lpalovsky is using in SDAF modules to check IP addresses format which could help here:

First: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/sles4sap/sdaf_library.pm#L19

And then: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/sles4sap/sdaf_library.pm#L398